mkosi/backend.py

```diff
@@ -506,6 +506,7 @@ class MkosiConfig:
     skeleton_trees: List[Path]
     clean_package_metadata: Union[bool, str]
     remove_files: List[Path]
+    # Environment should not be used directly. Use MkosiState environment instead.
```

Comment:
Can we drop this comment and replace it with a general comment above the config class itself to always use the respective field from MkosiState if it exists?